R/build-news.R

```diff
@@ -282,6 +295,36 @@ tweak_news_heading <- function(x, versions, timeline) {
   invisible()
 }
 
+tweak_news_heading_bs4 <- function(x, versions, timeline) {
+  x %>%
+    xml2::xml_find_all(".//h1") %>%
+    xml2::xml_set_attrs(c("class" = "pb-2 mt-4 mb-2 border-bottom"))
```

Comment:
Is this the only difference? If so, I think it would be better to keep a single function.

Reply:
no but there are only two differences (I had actually hesitated at the time) so I made it one single function again.